buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -16,101 +21,273 @@
-    automaticModule("io.grpc:grpc-protobuf", "grpc.protobuf")
-    automaticModule("io.grpc:grpc-services", "grpc.services")
-    automaticModule("io.grpc:grpc-protobuf-lite", "grpc.protobuf.lite")
-    automaticModule("io.grpc:grpc-testing", "io.grpc.testing")
-
-    automaticModule("org.openjdk.jmh:jmh-core", "jmh.core")
-    automaticModule("org.openjdk.jmh:jmh-generator-asm", "jmh.generator.asm")
-    automaticModule("org.openjdk.jmh:jmh-generator-bytecode", "jmh.generator.bytecode")
-    automaticModule("org.openjdk.jmh:jmh-generator-reflection", "jmh.generator.reflection")
+javaModuleDependencies {
+    versionsFromConsistentResolution(":hedera-node:node-app")
 
-    automaticModule("org.jetbrains:annotations", "annotations")
+    moduleNameToGA.put("com.hedera.hashgraph.protobuf.java.api", "com.hedera.hashgraph:hedera-protobuf-java-api")
+    moduleNameToGA.put("com.hedera.pbj.runtime", "com.hedera.pbj:pbj-runtime")
+    moduleNameToGA.put("com.swirlds.base", "com.swirlds:swirlds-base")
+    moduleNameToGA.put("com.swirlds.cli", "com.swirlds:swirlds-cli")
+    moduleNameToGA.put("com.swirlds.common.test", "com.swirlds:swirlds-common-test")
+    moduleNameToGA.put("com.swirlds.config", "com.swirlds:swirlds-config-api")
+    moduleNameToGA.put("com.swirlds.config.impl", "com.swirlds:swirlds-config-impl")
+    moduleNameToGA.put("com.swirlds.test.framework", "com.swirlds:swirlds-test-framework")
+    moduleNameToGA.put("hamcrest.core", "org.hamcrest:hamcrest-core")
+    moduleNameToGA.put("io.grpc", "io.helidon.grpc:io.grpc")
+    moduleNameToGA.put("io.helidon.webserver.http2", "io.helidon.webserver:helidon-webserver-http2")
+    moduleNameToGA.put("io.netty.codec.http", "io.netty:netty-codec-http")
+    moduleNameToGA.put("io.netty.codec.http2", "io.netty:netty-codec-http2")
+    moduleNameToGA.put("io.netty.codec.socks", "io.netty:netty-codec-socks")
+    moduleNameToGA.put("io.netty.handler.proxy", "io.netty:netty-handler-proxy")
+    moduleNameToGA.put("org.apache.logging.log4j.slf4j", "org.apache.logging.log4j:log4j-slf4j-impl")
+    moduleNameToGA.put("org.bouncycastle.util", "org.bouncycastle:bcutil-jdk15on")
+    moduleNameToGA.put("org.eclipse.collections.api", "org.eclipse.collections:eclipse-collections-api")
+    moduleNameToGA.put("org.eclipse.collections.impl", "org.eclipse.collections:eclipse-collections")
+    moduleNameToGA.put("org.hamcrest", "org.hamcrest:hamcrest")
+    moduleNameToGA.put("org.hyperledger.besu.datatypes", "org.hyperledger.besu:besu-datatypes")
+    moduleNameToGA.put("org.hyperledger.besu.evm", "org.hyperledger.besu:evm")
+    moduleNameToGA.put("org.hyperledger.besu.internal.rlp", "org.hyperledger.besu.internal:rlp")
+    moduleNameToGA.put("org.hyperledger.besu.plugin.api", "org.hyperledger.besu:plugin-api")
+    moduleNameToGA.put("org.mockito.junit.jupiter", "org.mockito:mockito-junit-jupiter")
+    moduleNameToGA.put("org.objenesis", "org.objenesis:objenesis")
```

Comment:
Ideally, this file is then the only place where we still find GA coordinates – From here we know which _Module Name_ maps to which _Coordinates_.